pkg/testutil/sitter.go

```diff
@@ -0,0 +1,24 @@
+package testutil
+
+import (
+	sitter "github.com/smacker/go-tree-sitter"
+)
+
+// FindNodeByContent returns a node whose content is the given content string. If there are multiple such nodes, this
+// will return one of them, but it's unspecified which.
+func FindNodeByContent(tree *sitter.Tree, content string) *sitter.Node {
```

Comment:
nit: if this used node instead of tree for input, the inner function wouldn't be needed

Reply:
I went for `Tree` just because it's a _little_ more obvious (at least to me) that it's a recursive/deep call. That is, finding a Node within a Tree has to be recursive, there's no other choice. Finding a Node from a Node _might_ be "return the given node, or nil`.

But maybe that's a silly enough alternative ("why wouldn't you just return bool?") that people won't assume it, or if they do assume it, they'll at least be 🤨'ed enough that they'll read the docs.